### DISCUSSION TITLE
Cleanup of temp files in temp directory?

### DISCUSSION THREAD

**Discussion by Hecatron** [copier-org/copier#708] [Q&A]
I'm noticing whenever I run copier I end up with a new directory created under
`C:\Users\username\AppData\Local\Temp`
such as `copier.vcs.clone.sxn86nzf`

This is the case regardless of running against a directory with a template in or a github repo
My question is should copier be cleaning up after itself after it finishes running?

**Comment by yajo** [comment]
It's an issue: #709

**Reply by Hecatron** [reply]
Yep I filed the issue after I raised the above
I noticed it only happened with directories associated as a git repo
